src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -1,16 +1,24 @@
-import * as React from 'react';
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
 import { FormattedMessage } from 'react-intl';
+import ItemList from './ItemList';
 import messages from '../messages';
 import PopupBase from '../PopupBase';
+import { Collaborator } from '../../../@types';
 import { Options, PopupReference } from '../Popper';
-
 import './PopupList.scss';
 
 export type Props = {
+    activeItemIndex?: number;
+    itemRowAs?: JSX.Element;
+    items: Collaborator[];
```

Comment:
Could this component be used for things other than collaborators? Is the object type of the array meaningful?